frontend/cs/scs/builder.go

```diff
@@ -686,3 +697,39 @@ func (builder *builder) ToCanonicalVariable(v frontend.Variable) frontend.Canoni
 		return term
 	}
 }
+
+func (builder *builder) storeWireLocation(a, b int) {
+	for i, v := range []int{a, b} {
+		if _, ok := builder.mRangecheckInterleave[i][v]; !ok {
+			builder.mRangecheckInterleave[i][v] = builder.cs.GetNbConstraints()
```

Comment:
Some notes: here we never capture the witness wires.

Reply:
Added optional argument to add dummy gates for witness wires.